docs/repositories.md

```diff
@@ -0,0 +1,61 @@
+# Repository Guidelines
+
+This document attempts to outline a structure for creating and associating
+GitHub repositories with the OpenSSF Scorecard project.
+<!-- TODO: It also describes how and when repositories are removed. -->
+
+<!-- TODO: Do we need a separate issue template for these requests? e.g.,
+Requests for creating, transferring, modifying, or archiving repositories can be made by [opening a request](https://github.com/ossf/scorecard/issues/new/choose) against this repository.
+-->
+
+- [Approval](#approval)
+- [Requirements](#requirements)
+- [Donated repositories](#donated-repositories)
+  - [Copyright headers](#copyright-headers)
+- [Attribution](#attribution)
+
+## Approval
+
+New repositories require approval from the OpenSSF Scorecard Steering Committee.
+
+## Requirements
+
+The following requirements apply to all OpenSSF Scorecard repositories:
+
+- Must be identified in the OpenSSF Scorecard project documentation
+- Must reside in the [OpenSSF GitHub organization](https://github.com/ossf)
+- Must utilize the topic [`openssf-scorecard`](https://github.com/topics/openssf-scorecard) (ref: [managing topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics))
```

Comment:
we should mark our other existing repos with this topic

Reply:
Done!